src/main/java/org/monarchinitiative/fhir2hpo/loinc/DefaultLoinc2HpoAnnotation.java

```diff
@@ -141,7 +178,7 @@ private HpoTermWithNegation getHpoTermForInternalCode(Loinc2HpoCodedValue code)
 			throws ConversionException {
```

Comment:
The `throws` declaration should probably match the `UnmappedInternalCodeException`.